src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -667,10 +663,9 @@ def build(self, notification_uuid: str | None = None) -> SlackBlock:
             blocks.append(self.get_suggested_assignees_block(suggested_assignees))
 
         # add suspect commit info
-        if event_for_tags:
```

Comment:
is there a reason this was taken out?

Reply:
we only need to fetch suspect commit from the group which is already passed in